src/web/src/layouts/bank/pages/accounts/logs/components/LogsTableSkeleton.tsx

```diff
@@ -1,11 +1,10 @@
-import React from 'react';
 import { Skeleton } from '@/components/ui/skeleton';
-import locales from '@/locales';
+import React from 'react';
 
 const LogsTableSkeleton: React.FC = () => {
   return (
     <div className="flex flex-col gap-2">
-      {Array.from({ length: 6 }).map((_, index) => (
+      {Array.from({ length: 6 }).map((_) => (
         <Skeleton className="h-[4.875rem] w-full"></Skeleton>
```

Comment:
this Skeleton component needs a `key` prop.

Reply:
fixed.